### PR TITLE
[FAB-134] WS Protobuf

### PR DESCRIPTION
**Pull request by daystram**
Migrate string JSON WS messages to binary Protobuf messages as an attempt to reduce network and CPU load on server and client, thus a better response time and horizontal scaling.

- [x] Create protos
- [x] Migrate fableous-fe to use protobufs
- [x] Migrate fableous-be to use protobufs
- [x] Benchmark before and after change
- [x] Update lint and build pipelines
- [x] Update deployment
- [x] Async WS handlers

### CODE REVIEW

File: fableous-fe/src/components/canvas/Canvas.tsx [hunk 1]
```diff
@@ -582,84 +582,84 @@ const Canvas = forwardRef<ImperativeCanvasRef, CanvasProps>(
...
+              placePaint(
+                x1,
+                y1,
+                x2,
+                y2,
+                msg.paint?.color || "#000000ff",
+                width || 8
+              );
+              if (process.env.NODE_ENV === "development")
+                console.log(
```
Comment (nicolauscg) [inline fableous-fe/src/components/canvas/Canvas.tsx:629]: is keeping this intentional?

Comment (daystram) [reply fableous-fe/src/components/canvas/Canvas.tsx:629]: Yes, we'll leave that here

**Review by nicolauscg — COMMENTED** [review]
partially done reviewing, managed to run it on localhost so all good.

when profiling before and after using pb, do we want to document it somewhere like in confluence? interested in how u did it and how much performance is gained

**Comment by daystram** [comment]
> partially done reviewing, managed to run it on localhost so all good.
> 
> when profiling before and after using pb, do we want to document it somewhere like in confluence? interested in how u did it and how much performance is gained

The method is nowhere scientific and purely empirical but I'll get it documented

**Review by nicolauscg — APPROVED** [review]

**Pull request merged**